src/connect.js

```diff
@@ -1,7 +1,12 @@
 import Sequelize from 'sequelize';
+import { log } from './logger';
 
 export const sequelize = new Sequelize('node_js_mp_2019_q4', 'postgres', 'postgres', {
   host: 'localhost',
   port: 5432,
   dialect: 'postgres',
 });
+
+sequelize.sync()
```

Comment:
Для домашки ОК. Для прода — за такое нужно бить по рукам. Синхронизация изменения моделей на живой продовой базе должна быть отключена и вместо этого должны быть использованы миграции.

Почему так:
1. Миграция может проходить _долго_. Например, полчаса или час — особенно если нужно перестроить какие-то индексы или материализованные представления. Всё это время сервис будет недоступен. Если у тебя в `.then` будет запускаться экспресс с liveness probe, то оркестратор будет принимать под за мёртвый и тушить его, потом переподнимать и всё повторится. Не далее как вчера на моем новом проекте команда влетела в это поведение со всех ног. Не надо так делать.
2. Миграции проще контролировать — накатывать и откатывать.
3. Миграции могут сразу преобразовать данные в изменяемых колонках. В ORM такое поведение обычно реализовать сложно или невозможно.

Reply:
Да, это определенно плохая практика. Тот же TypeORM вот так пишет об этом в своем ридми (выделение моё):

> `synchronize` - Indicates if database schema should be auto created on every application launch. **Be careful with this option and don't use this in production - otherwise you can lose production data.** This option is useful during debug and development. As an alternative to it, you can use CLI and run schema:sync command. Note that for MongoDB database it does not create schema, because MongoDB is schemaless. Instead, it syncs just by creating indices.

На всякий случай — для домашнего проекта или там тестового задания этого может быть более чем достаточно. Но не в проде.

Reply:
А тогда вопрос, как проверять, что коннект к базе произошел?

Reply:
Например, установкой коннекта без использования ORM. Или использовать подход, описанный в официальной доке: https://sequelize.org/master/manual/getting-started.html#test-the-connection